spec/support/features/steps/cases/data_request_steps.rb

```diff
@@ -1,3 +1,27 @@
+def record_a_data_request_of_cctv(checked_case, request_values)
```

Comment:
Are either of these methods being used?

Reply:
The request_values method is used in spec/features/cases/offender_sar/add_data_requests_spec.rb.
The other method checked_case isn't used